### ISSUE TITLE
stream

### ISSUE THREAD

**Issue by yoshuawuyts** [sheetify/sheetify#13]
Be able to do this:
```js
const sheetify = require('sheetify')
const http = require('http')

http.createServer((req, res) => {
  const s = sheetify(__dirname + '/index.css')
  s.bundle().pipe(res)
}).listen()

**Comment by yoshuawuyts** [comment]
Probably want to stay consistent with the [browserify api](https://github.com/substack/node-browserify#bbundlecb) and turn the cb into a buffer:
```js
sheetify(__dirname + '/index.css').bundle((err, buf) => {
  const css = buf.toString()
})

**Comment by yoshuawuyts** [comment]
Closed in 2863a59
